cmd/node/main_test.go

```diff
@@ -76,7 +76,7 @@ func getSmartInfo(serialNumber string) (*http.Response, error) {
 	var err error
 
 	for count := 0; count < maxRetries; count++ {
-		resp, err = http.Get("http://" + tMetricsAddress + tSmartPath + "/" + serialNumber)
+		resp, err = http.Get("http://" + tMetricsAddress + tSmartPath + "/drive/v1/" + serialNumber)
```

Comment:
shouldn't' it be /v1/drive/ instead of /drive/v1

Reply:
done